pom.xml

```diff
@@ -251,12 +251,6 @@
       <version>${truth.version}</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>com.google.truth</groupId>
-      <artifactId>truth</artifactId>
-      <version>${truth.version}</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
Why are we removing this dependency?

Reply:
@sabhyankar  It's a duplicate dependency. It was giving warnings will running the code.

Reply:
Dont remote anything that we didnt add in this PR :)